app/uk/gov/hmrc/claimvatenrolmentfrontend/controllers/CaptureBox5FigureController.scala

```diff
@@ -46,7 +48,8 @@ class CaptureBox5FigureController @Inject()(mcc: MessagesControllerComponents,
             _ => Ok(view(CaptureBox5FigureForm.form, routes.CaptureBox5FigureController.submit(journeyId)))
           }
         case None =>
-          throw new InternalServerException("Internal ID could not be retrieved from Auth")
+          errorLog(s"Internal ID could not be retrieved from Auth")
+          Future.successful(InternalServerError(errorHandler.internalServerErrorTemplate))
```

Comment:
The None on line 50 is from the authorised().retrieve(internalId) call. We would still want this to throw an error as this would mean something as gone wrong with auth. You need to handle a None returned by journeyService.retrieveJourneyConfig instead. I think it would be best to return a NotFound or a BadRequest error with the internalServerErrorTemplate because an InternalServerError will still return a 500 and trigger a pager duty

This needs to be updated on the other controllers too